docs/administration/configuration/configuring-private-docker-registries/index.md

```diff
@@ -0,0 +1,49 @@
+# Configuring private docker registries
+
+REANA is able to execute workflows which make use of Docker images hosted on private Docker image registries.
+You will just need to save the necessary credentials in Kubernetes secrets, and then instruct REANA to pass these secrets to each job's pod.
+
+## Obtaining the credentials
+
+First of all, you will need to obtain the authentication credentials, and how to do so varies depending on which registry you are using.
+One possibility would be to create a special service account to access the registry.
+As an example, the [REANA account](https://gitlab.cern.ch/reana) is used to authenticate to the private registry provided by CERN's GitLab instance, as shown in [Private Docker registries](../../../advanced-usage/access-control/private-docker-registries/#cern-gitlab).
+
+## Configuring Kubernetes and REANA
+
+After obtaining the credentials, you will have to save them in a Kubernetes secret, as explained in the related Kubernetes documentation page [Pull an Image from a Private Registry](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/).
+In particular, this is how you would create the secret from an already existing Docker configuration file, after running `docker login`:
+
+```{ .console .copy-to-clipboard }
+$ kubectl create secret generic <your-credentials-secret> \
+    --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
+    --type=kubernetes.io/dockerconfigjson
+```
+
+As an alternative, you can create the secret by providing the username and password needed to authenticate to the private registry:
+
+```{ .console .copy-to-clipboard }
+$ kubectl create secret docker-registry <your-credentials-secret> \
+    --docker-server=<your-registry-server> \
+    --docker-username=<your-name> \
+    --docker-password=<your-password> \
+    --docker-email=<your-email>
+```
+
+You should then set the [`components.reana_workflow_controller.environment.IMAGE_PULL_SECRETS`](https://github.com/reanahub/reana/tree/master/helm/reana) Helm value to the name of the secret, which in the above examples is `<your-credentials-secret>`:
+
+```yaml
+components:
+  reana_workflow_controller:
+    environment:
+      IMAGE_PULL_SECRETS: "<your-credentials-secret>"
+```
+
+If you would like to configure multiple private docker registries, you should create one secret for each of them. Then, you have to specify all the secrets in the `components.reana_workflow_controller.environment.IMAGE_PULL_SECRETS` Helm value, separating the names with commas:
+
+```yaml
+components:
+  reana_workflow_controller:
+    environment:
+      IMAGE_PULL_SECRETS: "<your-first-secret>,<your-second-secret>"
```

Comment:
```suggestion
      IMAGE_PULL_SECRETS: "<your-first-secret-name>,<your-second-secret-name>"
```

What do you think? Adding "name" might avoid that the users think they should type the secret itself rather than the name with which they added it to Kubernetes. The same goes for the example above with `<your-credentials-secret>`

Reply:
I would call them `mydockerhubsecret`, `myghrcsecret`, etc. explaining that this secret is for Docker Hub, that secret for GHCR.  (Just an example.)

Basically the idea is to create a scenario and provide concretely usable example that can be also "copy-pasted" and amended, instead of providing just a theoretical description. Hence the use of `john.doe@example.org` etc.

(See also the comment about `<foo>` syntax above.)

Reply:
I have updated the examples to refer to the CERN GitLab registry, but if you prefer to use a more known registry I can change to DockerHub